common/src/main/java/io/netty/util/internal/ObjectPool.java

```diff
@@ -50,6 +50,7 @@ public interface Handle<T> {
      *
      * @param <T> the type of the pooled object
      */
+    @Deprecated
```

Comment:
please add deprecation comments. also, can the whole class be deprecated?

Reply:
`ObjectPool` has `public interface Handle<T>`, which is still used. We can extract it, but that's better to do via a separate RP.